package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Rocket.Chat",
-	"version": "2.4.0-develop",
```

Comment:
Please don't commit your `package-lock.json` file

Reply:
Sorry, I didn't know about that. I'll make the necessary changes.